TidyPlatesInternal/TidyPlatesCore.lua

```diff
@@ -2084,7 +2088,18 @@ do
     local db = Addon.db.profile.settings
 
     -- Healthbar
-		SetAnchorGroupObject(visual.healthbar, style.healthbar, extended)
+    local healthbarOpts = {
+      x = style.healthbar.x,
+      y = style.healthbar.y,
+      width = style.healthbar.width,
+      height = style.healthbar.height
+    }
+
+    if extended.unit.reaction == 'FRIENDLY' then
+      healthbarOpts.width = style.healthbar.widthFriendly
+      healthbarOpts.height = style.healthbar.heightFriendly
+    end
+    SetAnchorGroupObject(visual.healthbar, healthbarOpts, extended)
```

Comment:
I think it would be enought to change the style parameter (healthbarOpts) only if extended.unit.reaction == 'FRIENDLY', so basically only create the array in the if-part. Not sure though. Need to think about this a bit more.